docs/spfx/library-component-tutorial.md

```diff
@@ -45,7 +45,7 @@ localization_priority: Priority
  
 1. Open the solution in your favorite editor and navigate to ```src\corporateLibrary\CorporateLibraryLibrary.ts```
 
-1. You will notice that a default method, ```name()``` has been created. Rename this method as follows:
```

Comment:
Inline code references should be surrounded with a single backtick, not triple. Triple backticks are used to fence code blocks, not inline code.